app/helpers/react_components/student/open_editor_button.rb

```diff
@@ -4,6 +4,9 @@ class OpenEditorButton < ReactComponent
       initialize_with :exercise
 
       def to_s
+        # TODO: What should we do with this?
```

Comment:
I don't think this is even possible, see that `user_track` calls into the [`for` method](https://github.com/exercism/website/blob/main/app/models/user_track.rb#L22) which then uses `find_by!`. I think this means that an exception would be raised if the track could not be found, thus not needing to add this check.

Reply:
Actually, `for` returns a `UserTrack.external`. `for!` is what raises.

But Erik is right, it will never be blank, so this should be return if `user_track.external?`

Add a test pls! :)